ui/app/components/masked-input.js

```diff
@@ -12,13 +12,16 @@ import autosize from 'autosize';
  *  @value={{attr.options.defaultValue}}
  *  @placeholder="secret"
  *  @allowCopy={{true}}
+ *  @onChange={{action "someAction"}}
  * />
  * ```
  *
  * @param [value] {String} - The value to display in the input.
  * @param [placeholder=value] {String} - The placeholder to display before the user has entered any input.
  * @param [allowCopy=null] {bool} - Whether or not the input should render with a copy button.
  * @param [displayOnly=false] {bool} - Whether or not to display the value as a display only `pre` element or as an input.
+ * @param [onChange=false] {Function|action} - A function to call when the value of the input changes.
```

Comment:
yay, storybook updates in action! thank you for remembering to do this!

Reply:
oh also, should the default value of `onChange` be null instead of false?

Reply:
oops, copy and paste error - the default is an empty fn, I'm not sure how to express that here... 🤔

Reply:
yeah, i noticed that. i think you could try putting an empty fn in here and if that doesn't work, i think you could just omit the default value and list it like so:
`* @param [onChange] {Function|action} - A function to call when the value of the input changes.`

Reply:
Went with `Function.prototype` since all code gets escaped and garbled. Function.prototype is also an empty fn, so I think that works for now. Would be nice to do `() => {}` or similar instead though.